2gp-demo/config/subscriber-scratch-def.json

```diff
@@ -0,0 +1,13 @@
+{
+  "orgName": "PLP Dev Preview (2GP Subscriber)",
```

Comment:
i see now there are multiple scratch defs. how and where are these used as different files?

Reply:
The "subscriber" scratch def uses the Enterprise template instead of the Developer template.  This is a more realistic environment for testing and is the recommended best practice.